### PR TITLE
Fix fatal error on initialization

### PR DESCRIPTION
**Pull request by udev**
fixing fatal error related to settings being used before they are initialized

### CODE REVIEW

**Comment by jmtobias** [comment]
Can you update the version number in the README and the header comment in chargeio-for-woocommerce.php file?

**Comment by udev** [comment]
Changes added per @jmtobias 
The plugin appears to be working now 👍 

image of receipt

**Comment by jwsparrow** [comment]
A couple notes: We can whack Domenic from the author tags throughout. I'm really not a fan of author tags, especially on source that ends up being accessed by external parties -- I should have done this already. Also, we can replace any reference to "ChargeIO" and "ChargeIO Gateway" with "AffiniPay" and "AffiniPay Payment Gateway". We're slowly getting rid of all the ChargeIO naming across our solution. I'm not sure of the impact of making this change on the class names (the "cio" prefix, "chargeio-for-woocommerce", etc), but at least we can fix it in the readme and the file comments.

**Comment by udev** [comment]
@jwsparrow would it make sense to break the mentioned changes out into a separate "cleanup" PR to keep the history clean?

**Comment by udev** [comment]
Additionally we need to move on this fix as it might directly relate to issue #1

File: classes/class-cio4wc_gateway.php [hunk 1]
```diff
@@ -375,9 +375,10 @@ public function payment_fields() {
         cio4wc_get_template( 'payment-fields.php' );
 
         // Output WooCommerce 2.1+ cc form
-        $this->credit_card_form( array(
-            'fields_have_names' => false,
-        ) );
+//        $this->credit_card_form( array(
```
Comment (jmtobias) [inline classes/class-cio4wc_gateway.php]: Should we remove these commented out lines if they are not needed?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # ChargeIO for WooCommerce
 
-Plugin version: 1.2
+Plugin version: 1.3
 
 Requires at least:
 - WordPress 3.8.0
```
Comment (jmtobias) [inline README.md]: We should probably update the minimum WordPress version to 4.4 to match the WooCommerce requirements.

Comment (udev) [reply README.md]: Good catch :)

**Review by jmtobias — APPROVED** [review]
Looks Good to Me

**Review by jmtobias — APPROVED** [review]

**Review by jmtobias — APPROVED** [review]
Looks good to me

**Pull request merged**